### ISSUE TITLE
您好，首先感谢您的工作！我用了您上传到百度云盘的数据集，可为什么训练参数不一样

### ISSUE THREAD

**Issue by SUZILI7** [CrossmodalGroup/HREM#12]
2024-12-18 17:25:48,583 Namespace(base_loss='trip', batch_size=256, bert_path='data/bert-base-uncased', cross_loss=1, data_path='data/', dataset='coco', decay_rate=0.1, dropout=0.1, embed_size=256, gnn_loss='trip', gpu_id=0, grad_clip=2.0, graph_lr_factor=1.0, img_dim=2048, learning_rate=0.0005, log_step=200, logger_name='runs/coco_test', lr_schedules=[15, 25], margin=0.2, mask_repeat=1, mask_weight=1.0, max_violation=False, model_name='runs/coco_test', multi_gpu=0, nhead=16, no_imgnorm=False, no_txtnorm=False, norm_input=1, num_epochs=30, num_layers_enc=1, reg_loss_weight=10, residual_weight=0.8, save_results=1, size_augment=1, threshold=0.5, topk=10, val_step=500, vse_mean_warmup_epochs=1, warmup=8000, workers=8)
2024-12-18 17:26:14,838 Number of images for train-set: 113287
2024-12-18 17:26:20,044 Log saving path: runs/coco_test
2024-12-18 17:26:20,044 Models saving path: runs/coco_test
2024-12-18 17:26:20,045 image encoder trainable parameters: 1.16M
2024-12-18 17:26:20,047 txt encoder trainable parameters: 104.98M
2024-12-18 17:26:20,047 criterion trainable parameters: 0.38M

**Comment by darkpromise98** [comment]
你设置了 embed_size=256，训练参数就会少一些（默认设置为embed_size=1024）

**Comment by SUZILI7** [comment]
> 你设置了 embed_size=256，训练参数就会少一些（默认设置为 embed_size=1024）

好的，非常感谢！
